library/src/test/java/com/getstream/sdk/chat/MarkDownTextTest.java

```diff
@@ -35,13 +35,22 @@ void containStrikeThroughTextTest(){
 
     @Test
     void containHeaderTextTest(){
-        String text = "# TestMarkDown";
+        String text = "# h1";
         assertTrue(StringUtility.containsMarkdown(text));
 
-        text = "## TestMarkDown";
+        text = "## h2";
         assertTrue(StringUtility.containsMarkdown(text));
 
-        text = "### TestMarkDown";
+        text = "### h3";
+        assertTrue(StringUtility.containsMarkdown(text));
+
+        text = "#### h4";
+        assertTrue(StringUtility.containsMarkdown(text));
+
+        text = "##### h5";
+        assertTrue(StringUtility.containsMarkdown(text));
+
+        text = "###### h6";
```

Comment:
@andriizhumela , I have updated `containHeaderTextTest()`
in case `text = "###### h6"`, doesn't work.

![image](https://user-images.githubusercontent.com/11132956/70152023-bc721300-16ac-11ea-8e30-c838bf240d81.png)

Please refer this test link: https://markdown-it.github.io/
And then SDK doesn't provide all Markwon features.
https://github.com/GetStream/stream-chat-android#markdown-support

When you are testing, please consider above.

Reply:
@adrian09h I expanded the test suite and added a missing character check.

Reply:
@andriizhumela , currently we don't provide all markwon features.

![image](https://user-images.githubusercontent.com/11132956/70223736-65237f80-174c-11ea-9dde-a08c46d2ba1b.png)

However you have added `containTypographicTextTest` and updated `containsMarkdown`.

SDK doesn't provide obviously `Typographic` feature currently.

Can you please remove the code for non-provided markwon feature from `Test` and `containsMarkdown`?